nixos/modules/services/networking/dropbox.nix

```diff
@@ -0,0 +1,73 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.dropbox;
+
+in
+
+{
+  ###### interface
+
+  options.services.dropbox = {
+
+    enable = mkEnableOption ''
+      Whether to enable the dropbox service globally.
+    '';
+
+    install = mkEnableOption ''
+      Whether to install a user service for the dropbox daemon. Once
+      the service is started, the Dropbox desktop app is available.
+
+      The service must be manually started for each user with
+      "systemctl --user start dropbox" or globally through
+      <varname>services.dropbox.enable</varname>.
+    '';
+
+    openFirewall = mkEnableOption ''
+      Open ports in the firewall for Dropbox.
+
+      UDP: 17500
+      TCP: 17500
+    '';
+
+  };
+
+  ###### implementation
+
+  config = mkIf (cfg.enable || cfg.install) {
+
+    environment.systemPackages = with pkgs; [ dropbox-cli ];
+
+    systemd.user.services.dropbox = {
+      description = "Dropbox daemon";
+      after = [ "xembedsniproxy.service" ];
```

Comment:
Huh?

Reply:
@peterhoeg can you comment on the dependency on `xembedsniproxy.service`? What is its purpose?

Reply:
It used to be the only way to have dropbox show a trayicon - I have no idea if that is still the case as I'm not using dropbox any more.

Reply:
Looks like some kind of plasma thing: https://github.com/davidedmundson/xembed-sni-proxy